pxr/base/tf/denseHashSet.h

```diff
@@ -55,7 +53,12 @@ template <
     class    EqualElement  = std::equal_to<Element>,
     unsigned Threshold = 128
 >
-class TfDenseHashSet
+class TfDenseHashSet :
```

Comment:
Same comment here; we don't want the DenseHashSet inheriting from these function types.